### PR TITLE
add enterprise-cli-subcommands package

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by vespian** [comment]
LGTM, but tests are failing :(

File: repo/packages/E/enterprise-cli-subcommands/0/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packagingVersion": "2.0",
```
Comment (BenWhitehead) [inline repo/packages/E/enterprise-cli-subcommands/0/package.json]: `packagingVersion` needs to be `3.0` if a marathon.json.mustache isn't provided.

File: repo/packages/E/enterprise-cli-subcommands/0/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "name": "enterprise-cli-subcommands",
+  "version": "0.1.0",
+  "maintainer": "support@mesosphere.io",
+  "description": "Enterprise only cli subcommands",
+  "tags": [ "mesosphere", "enterprise", "subcommand" ]
```
Comment (BenWhitehead) [inline repo/packages/E/enterprise-cli-subcommands/0/package.json]: please add `"minDcosReleaseVersion": "1.8"` to the package.

**Comment by vespian** [comment]
LGTM!

**Pull request merged**